### PR TITLE
[WIP] Updated evergreene's slide

### PR DESCRIPTION
**Pull request by evergreene**
Cheerio,

cc: @ukbill enlighten yourself  please

### CODE REVIEW

**Review by ukbill — APPROVED** [review]

File: _posts/0000-01-05-evergreene.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Spotted Puss
```
Comment (ukbill) [inline _posts/0000-01-05-evergreene.md:6]: No comment for fear of being had up in front of HR

Comment (ukbill) [inline _posts/0000-01-05-evergreene.md:6]: @evergreene - we need to start a review about this one.

File: _posts/0000-01-05-evergreene.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (ukbill) [inline _posts/0000-01-05-evergreene.md]: Do we want to change this line too?

File: _posts/0000-01-05-evergreene.md (hunk 1 above)
Comment (evergreene) [reply _posts/0000-01-05-evergreene.md:6]: Done

**Review by ukbill — APPROVED** [review]

File: _posts/0000-01-05-evergreene.md (hunk 2 above)
Comment (evergreene) [reply _posts/0000-01-05-evergreene.md]: groovy

**Pull request merged**